R/rules.R

```diff
@@ -251,18 +279,24 @@ sv_integer <- function(message = "An integer is required",
 #' 
 #'   iv$add_rule("count", sv_between(10, 10000))
 #'   iv$add_rule("count", ~if (. <= 0) "A positive value is required")
-#'
 #' })
+#' 
 #' @export
 sv_between <- function(left,
                        right,
                        inclusive = c(TRUE, TRUE),
+                       message = NULL,
                        allow_na = FALSE,
-                       allow_nan = FALSE,
-                       message = NULL) {
+                       allow_nan = FALSE) {
+  force(left)
+  force(right)
+  force(inclusive)
+  force(message)
   force(allow_na)
   force(allow_nan)
-  force(message)
+  
+  # TODO: allow for check of multiple values with `allow_multiple`
+  # TODO: `message` should have the message string 
```

Comment:
If we do this, and call glue_data_safe on `message` every time, then change the parameter name to `message_fmt` instead (and make sure the docs indicate that `message_fmt` can contain {left} and {right}).

Reply:
This is now done for every `message` that could use string interpolation. Documentation updated as well.